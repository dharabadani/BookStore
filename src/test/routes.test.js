import React from 'react';
import renderer from 'react-test-renderer';
import routes from '../routes';

test('Test Routes', () => {
    const routes = renderer.create(<routes />);
    expect(routes.toJSON()).toMatchSnapshot();
});