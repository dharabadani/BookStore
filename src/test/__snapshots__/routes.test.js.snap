// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Routes 1`] = `<routes />`;
